engine/apps/labels/utils.py

```diff
@@ -42,19 +42,28 @@ def get_associating_label_model(obj_model_name: str) -> typing.Type["AssociatedL
     return label_model
 
 
-def is_labels_feature_enabled(organization) -> bool:
-    """
-    Checks if labels feature enabled for all organizations (FEATURE_LABELS_ENABLED_FOR_ALL).
-    If not, checks if current organization's grafana org_id is in the list of organizations labels feature enabled for
-    (FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS)
-    """
-    logger.info(
```

Comment:
removing logging here as this function will be called upon alert creation (which is potentially very frequent)